Source/Core/AttributeCompression.js

```diff
@@ -303,8 +304,9 @@ define([
         //>>includeEnd('debug');
 
         var temp = compressed / 4096.0;
-        result.x = Math.floor(temp) / 4096.0;
-        result.y = temp - Math.floor(temp);
+        var xZeroTo4095 = Math.floor(temp);
```

Comment:
The reason I did it this way was to mirror what was done in the shader which doesn't have any bitwise operations, but its up to you.